server/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -276,6 +280,9 @@ public void writeTo(StreamOutput out) throws IOException {
     @Override
     public final XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject();
+        if (builder.getCompatibleMajorVersion() == Version.V_7_0_0.major) {
+            builder.field(TYPE_FIELD_NAME, SINGLE_MAPPING_TYPE);
```

Comment:
I think you can just do the following, I've never had to wrap a string inside a `Text` object:

```
builder.field(TYPE_FIELD_NAME, MapperService.SINGLE_MAPPING_NAME);
```

Reply:
will do